src/frontend/sharedComponents/PracticeMode.tsx

```diff
@@ -0,0 +1,68 @@
+import * as React from "react";
+import { useContext } from "react";
+import { defineMessages, FormattedMessage } from "react-intl";
+import { View, StyleSheet, Text } from "react-native";
+import MaterialIcons from "react-native-vector-icons/MaterialIcons";
+import ConfigContext from "../context/ConfigContext";
+
+import { DARK_ORANGE, WHITE } from "../lib/styles";
+
+const m = defineMessages({
+  title: {
+    id: "sharedComponents.PracticeMode.title",
+    defaultMessage: "Practice Mode",
+    description: "Indicated to user that they are on practice mode",
+  },
+});
+
+interface PracticeModeProps {
+  children: React.ReactNode;
+}
+
+export const PracticeMode = ({ children }: PracticeModeProps) => {
+  const [config] = useContext(ConfigContext);
+
+  //TODO change how we determine whether we are in practice mode or not
+  if (config.metadata.name === "mapeo-default-settings") {
+    return (
+      <View style={styles.container}>
+        {children}
+
+        {config.practiceModeBar && (
+          <View style={styles.bottomBar}>
+            {/* Placeholder icon */}
+            <MaterialIcons name="highlight" color={WHITE} size={30} />
```

Comment:
I've been using this icon since it's somewhat close (you'll have to apply a transform to rotate it)

<img width="116" alt="image" src="https://user-images.githubusercontent.com/18542095/131360300-0a3c780c-e5e4-46f0-b863-61bf56d20f76.png">

Reply:
did you know this existed: https://oblador.github.io/react-native-vector-icons/

Sabella just introduced it to me!

Reply:
haha yeah that's what i've been referencing

Reply:
ah wait nvm i've been using: https://react-icons.github.io/react-icons/icons?name=md

Reply:
Wow! Ive literally been going into each library and looking at icons that way.....